server.js

```diff
@@ -45,6 +45,13 @@ app.prepare().then(() => {
             : pathname)
       })
       res.end()
+    } else if (req.headers.host === 'err.dvc.org') {
```

Comment:
let's make a regular one - just `error`? not sure we need to be cryptic here

Reply:
Sure? I like that it's super short kind of line `man.dvc.org` helps with tiny URL, maybe easier to print on command line. It's not like "err" could be any other word :P but as you guys prefer. Very easy change obvi

Reply:
Please vote @pared @efiop 😬 (and feel free to change the code of course), and please let Ivan know so he sets up the DNS record for the chosen subdomain.

Reply:
I agree with @jorgeorpinel, that `err` is hard to misinterpret. Thats not that big of an issue, can be either way, if @shcheklein thinks `error` will be more explicit, lets go this way.
@efiop ?

Reply:
I agree with @shcheklein , `error` looks much more visually pleasing. I don't run into `err` very often (almost never, except in some internal code pieces), so it looks incomplete and weird to me.

Reply:
Ok sure as you guys prefer. I have no strong opinion against `error` 🙂

Reply:
I see this is addressed. Resolving.

Reply:
p.s. @shcheklein please setup the error.dvc.org DNS record when you can